build.gradle

```diff
@@ -4,110 +4,109 @@ plugins {
     id 'java-library'
     id 'maven-publish'
 
-    id 'com.diffplug.spotless' version '5.12.4'
-    id 'com.github.ben-manes.versions' version '0.36.0'
-    id 'com.jfrog.artifactory' version '4.28.1'
+    id 'com.diffplug.spotless' version '6.10.0'
+    id 'com.github.spotbugs' version '5.0.12'
+    id 'com.github.ben-manes.versions' version '0.42.0'
+    id 'com.jfrog.artifactory' version '4.28.2'
     // [dd 2021-05-17] Use of Spring dependency management generates invalid POM, making
     // it impossible to publish to mavenLocal. In order to make TCL testable, I have
     // removed this plugin and filled in specific version numbers. I also removed all
     // version numbers with `+` in them. Was:
     //     id 'io.spring.dependency-management' version '1.0.10.RELEASE'
-    id 'org.springframework.boot' version '2.6.6'
-    id 'ru.vyarus.quality' version '4.5.0'
+    id 'org.springframework.boot' version '2.7.3'
+    id 'ru.vyarus.quality' version '4.8.0'
 }
 
+// TODO(PF-1981) - Move version to settings.gradle
+group 'bio.terra'
+version '0.0.68-SNAPSHOT'
+
+// If true, search local repository (~/.m2/repository/) first for dependencies.
+def useMavenLocal = false
 repositories {
+    if (useMavenLocal) {
+        mavenLocal() // must be listed first to take effect
+    }
     mavenCentral()
     maven {
-        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release/'
+        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/'
```

Comment:
Hmm, are you sure about this? The artifactory libraries are not organized the way you might expect.

Reply:
this is based on artifacts in https://broadinstitute.jfrog.io/ui/repos/tree/General/libs-release-local
besides both libs-release-local & libs-release point to the same (libs-release-local)